src/userGuide/UserGuideFi.tsx

```diff
@@ -323,26 +323,25 @@ function UserGuideFi(): ReactElement {
           Voit halutessasi yhdistää eri tunnistautumistavat yhdeksi
           Helsinki-profiiliksi, jolloin voit tarkastella ja hallinnoida kaikkia
           tietojasi ja hyödyntämiäsi palveluja kerralla. Tämä onnistuu tekemällä
```

Comment:
Not a part of the PR, but this should be "luomalla" (create), not "tekemällä" (make/do).
```suggestion
          tietojasi ja hyödyntämiäsi palveluja kerralla. Tämä onnistuu luomalla
```

Reply:
Juuri näin.